src/QtUtils/include/QtUtils/ExecuteProcess.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef QT_UTILS_EXECUTE_PROCESS_H_
+#define QT_UTILS_EXECUTE_PROCESS_H_
+
+#include <absl/time/time.h>
+
+#include <QByteArray>
+#include <QObject>
+#include <QString>
+#include <QStringList>
+
+#include "OrbitBase/Future.h"
+#include "OrbitBase/Result.h"
+
+namespace orbit_qt_utils {
+
+constexpr absl::Duration kDefaultTimeout = absl::Seconds(5);
```

Comment:
This name is too generic for being top level (`orbit_qt_utils::kDefaultTimeout`). I suggest something like `kExecuteProcessDefaultTimeout`

Reply:
done